build.sh

```diff
@@ -53,7 +54,7 @@ while :; do
             CUSTOM_SOURCE_BUILT_PACKAGES="$(abspath "$CUSTOM_SOURCE_BUILT_PACKAGES")"
             shift
             ;;
-        -?|-h|--help)
+        "-?"|-h|--help)
```

Comment:
Unless `-?` is quoted, it means `-` followed by any single character. It takes over `-h`, and everything from `-a` to `-z` too.